packages/dashmate/src/listr/tasks/startNodeTaskFactory.js

```diff
@@ -6,6 +6,18 @@ const { Observable } = require('rxjs');
 
 const { NETWORK_LOCAL } = require('../../constants');
 
+const ensureLogFilePath = (logFilePath) => {
```

Comment:
I guess you want to create a function in `src/docker` and call it `ensureFileMountExists` so you can reuse it here and `startCoreFactory`